cmd/internal/server/handlers/fivetran_value_converters.go

```diff
@@ -214,3 +216,51 @@ func GetConverter(dataType fivetransdk.DataType) (ConverterFunc, error) {
 	}
 	return converter, nil
 }
+
+func GetEnumConverter(enumValues []string) (ConverterFunc, error) {
+	return func(value sqltypes.Value) (*fivetransdk.ValueType, error) {
+		parsedValue := value.ToString()
+		index, err := strconv.ParseInt(parsedValue, 10, 64)
+		if err != nil {
```

Comment:
In another PR, it would be good to replace this with a definitive "if not in copying phase, map to value" conditional instead. But this is a step in the right direction I think!

Reply:
Doing so will allow us to successfully parse cases where the user has integer ENUM values `enum("1,",2","3")`. This is not recommended by MySQL but exists. See ["Handling of Enumeration Literals"
](https://dev.mysql.com/doc/refman/8.0/en/enum.html)